### ISSUE TITLE
possibility of having a more precise soh?

### ISSUE THREAD

**Issue by cyrelec86** [openvehicles/Open-Vehicle-Android#139]
Hello the community I wanted to know if it was possible to have a soh with 2 decimal places? I am doing a study on the degradation of the 40 kw battery of the Léaf and I need a precise soh! do you think it is possible?
thank you in advance for your answer 😉

**Comment by glynhudson** [comment]
I don't think the SOH reading from the BMS is accurate enough to merit two decimal places! The SOH is a rough estimation of a chemical process, it's a very rough estimation at best. I wouldn't even consider looking at even one decimal place. The average SOH over a rolling 12-month period is a better measurement. Here's a 3yr graph os SOH of my 2014 Leaf, now with 90,000 miles. 

![Screenshot 2023-03-30 00 41 10](https://user-images.githubusercontent.com/758844/228691760-8dbfdbcf-25c3-4acd-aa68-7bff4b0304b5.png)

**Comment by dexterbg** [comment]
Correct. Adding decimal places suggests precision and accuracy where there is really none.

If you're interested in the raw value the vehicle provides, you can always do `metrics list soh`. Some vehicle adapters provide multiple indicators, check their documentation for custom metrics, but the "soh" filter will probably already include them.
